images.py

```diff
@@ -55,16 +58,38 @@ def _server_query(url, q):
     return doc
 
 
-# The Custom Search identifier
-_CX = "001858240983628375092:9aogptqla5e"
 # The Google API identifier (you must obtain your own key if you want to use this code)
 _API_KEY = ""
+_API_KEY_PATH = "resources/GoogleServerKey.txt"
+
+
+def _get_API_key():
+    """ Read Google API key from file """
+    global _API_KEY
+    if _API_KEY:
```

Comment:
Better: `if not _API_KEY: try [...]`, then end with `return _API_KEY` in all cases